src/promptflow/tests/sdk_cli_test/e2etests/test_flow_run.py

```diff
@@ -745,3 +745,13 @@ def test_system_metrics_in_properties(self, pf) -> None:
         assert FlowRunProperties.SYSTEM_METRICS in run.properties
         assert isinstance(run.properties[FlowRunProperties.SYSTEM_METRICS], dict)
         assert "total_tokens" in run.properties[FlowRunProperties.SYSTEM_METRICS]
+
+    # test image
+    def test_basic_image_flow_bulk_run(self, pf) -> None:
+        data_path = f"{FLOWS_DIR}/python_tool_with_image_input_and_output/image_inputs/inputs.jsonl"
+
+        run = pf.run(flow=f"{FLOWS_DIR}/python_tool_with_image_input_and_output", data=data_path)
+
+        local_storage = LocalStorageOperations(run)
+        detail = local_storage.load_detail()
+        print(detail)
```

Comment:
Assert detail's content? Are we using absolute path for pics?

Reply:
Updated.